src/components/Movie/MovieCard.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+import { Link } from "react-router-dom";
+
+const aStyle = {
+    textDecoration: 'none',
+    color: 'inherit'
+};
+export default class MovieCard extends Component {

```

Comment:
This MovieCard and MovieBanner cannot be the same component?

Reply:
MovieCard is the individual tile which appears as a search result. Whereas MovieBanner lists all the information about the movie in detail.